### PR TITLE
Update Lab2 solution

### PR DESCRIPTION
**Pull request by mrvee-qC-bee**
Update solution for lab 2

### CODE REVIEW

**Comment by mrvee-qC-bee** [comment]
@sumit-kale

File: solutions/lab_2/lab-2.ipynb [hunk 1]
```diff
@@ -0,0 +1,2231 @@
+{
```
Comment (0sophy1) [inline solutions/lab_2/lab-2.ipynb:1]: <pre class="ql-syntax" spellcheck="false">tr_depths = []
tr_gate_counts = []
tr_scores = []

approximation_degree_list = np.linspace(0,1,10)

for i in range(4):
    for j in approximation_degree_list:
        pm = generate_preset_pass_manager(backend=backend, optimization_level=i, approximation_degree=j, seed_transpiler=seed) 
    
        tr_depths.append(pm.run(qc).depth())
        tr_gate_counts.append(sum(pm.run(qc).count_ops().values()))
        tr_scores.append(scoring(pm.run(qc), backend))
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qiskit-community/ibm-quantum-challenge-2024/pull/56/discussion/#comment-1646388723'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='95'/>

**Comment by 0sophy1** [comment]
Thank Vishal, I added util.py and updated one code block. Thanks!

**Review by 0sophy1 — APPROVED** [review]
Code updated and util.py added. Ready to be merged

**Pull request merged**